MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -1725,6 +1726,7 @@ private void clrprf()
         {
             //Set prfptr to the base address of prfbuf
             Module.Memory.SetPointer("PRFPTR", Module.Memory.GetVariablePointer("PRFBUF"));
+            Module.Memory.SetByte(Module.Memory.GetVariablePointer("PRFBUF"), 0);
```

Comment:
maybe store the value from line 1728's GetVariablePointer and reuse here? Seems silly to repeat the same lookup you just did. 

As an future optimization, maybe it would make sense to cache all these common variable pointers so they don't have to be looked up

Reply:
`GetVariablePointers` returns a reference from a Dictionary which is a hash table lookup and close to an O(1) operation -- essentially the cache 👍

Reply:
grumble grumble O(1) is still less than O(2) ;-)

Can always be changed later